docs/rst/tutorials/core/deployment/deployment_tutorials.rst

```diff
@@ -21,3 +21,4 @@ For tutorials related to cloud or cloud-edge deployments, please refer :ref:`tut
     user_topics_multicast/user_topics_multicast
     keys/topic_keys
     keys/filtered_topic_keys
+    easy_mode/easy_mode
```

Comment:
Move the tutorial to be the first of these list 
https://vulcanexus--233.org.readthedocs.build/en/233/rst/tutorials/core/wifi/wifi_tutorials.html